lib/workers/repository/process/lookup/index.ts

```diff
@@ -262,7 +261,27 @@ export async function lookupUpdates(
       if (!currentVersion! && config.lockedVersion) {
         return res;
       }
+
       res.currentVersion = currentVersion!;
+      const currentVersionRelease = allVersions.find((v) =>
+        versioning.equals(v.version, currentVersion),
+      );
```

Comment:
```suggestion
      const currentVersionTimestamp = allVersions.find((v) =>
        versioning.equals(v.version, currentVersion),
      )?.releaseTimestamp;
```

.. and then some other code needs to change too